FitGoal/FitGoal/Singup&Singin/SocialMediaAuthenticator.swift

```diff
@@ -11,7 +11,7 @@ import Firebase
 import GoogleSignIn
 import FBSDKLoginKit
 
-class SocialMediaAuthenticator: NSObject, GIDSignInDelegate {
+class Authenticator: NSObject, GIDSignInDelegate {
```

Comment:
You changed the name fo the class but you forgot to change the name of the file.